@here/harp-mapview/lib/MapView.ts

```diff
@@ -2250,6 +2250,17 @@ export class MapView extends THREE.EventDispatcher {
         this.clearTileCache();
     }
 
+    /**
+     * Allows the user to block some areas of the screen.
+     * @note To clear, simply pass in an empty array.
+     * @param screenSpaceBoxes Boxes that are in pseudo screen space, i.e. the center of the screen
+     * is at 0,0, whilst the left top point is -width/2, -height/2 and the bottom right is width/2
+     * and height/2.
+     */
+    setScreenSpaceBoxes(screenSpaceBoxes: Math2D.Box[]) {
```

Comment:
I suggest to rename it to sth like. `setLabelAvoidanceArea` to describe what it is actually doing.

Reply:
Done, I have a similar method for the lines, so I changed it a bit.